src/Http/OctaneHandler.php

```diff
@@ -29,12 +31,14 @@ public function handleRequest(?HttpRequestEvent $event, ?Context $context): Http
             $response = OctaneClient::handle($request);
         }
 
-        if (! $response->headers->has('Content-Type')) {
-            $response->prepare($request);
-        }
+        $response->prepare($request);
+
+        $content = $response instanceof BinaryFileResponse
```

Comment:
I haven't tested `StreamedResponse`, should we support that too?

Reply:
Can't stream from Lambda afaik.